manifests/g/GNU/Solfege/3.23.4/GNU.Solfege.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: GNU.Solfege
+PackageVersion: 3.23.4
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerUrl: https://alpha.gnu.org/gnu/solfege/solfege-win32-3.23.4.exe
+  InstallerSha256: 0876E0D5EBF926E1DE399F264C3596F059EE2781CF09F0D6C42E652BF02DFAC7

```

```suggestion
  InstallerSha256: 0876E0D5EBF926E1DE399F264C3596F059EE2781CF09F0D6C42E652BF02DFAC7
  UpgradeBehavior: uninstallPrevious
```

Their documentation says a uninstall is required before an upgrade.